### ISSUE TITLE
Docker Desktop Dev Environments: .devcontainer.json Integration

### ISSUE THREAD

**Issue by marchesir** [docker/roadmap#331]
**Tell us about your request**
When DD (Docker Desktop) dev enviroment is created if .devcontainer.json allready exists it is ignored.
As many git repos above all now have .devcontainer.json integration, e.g. GitHub codespaces, local vscode, cli docker.  All are controlled by the .devcontainer.json file: https://code.visualstudio.com/docs/remote/devcontainerjson-reference.  It would make the DD dev envirionments perfect, but when created the .devcontainer.json is ignored even if present.

**Which service(s) is this request for?**
DD dev environments.

**Tell us about the problem you're trying to solve. What are you trying to do, and why is it hard?**
Im trying to use DD dev environments but id like them to be aware of pre-existing .devcontainer.json and not ignore.  My objective is not to touch local, in that all tooling is isolated into containers, some in git repo others just local but DD should if .devcontainer.json found boot that config not create new  based on language or default.

**Are you currently working around the issue?**
Not sure if there is a workaround, just use vscode/docker directly and not DD dev environments which is a shame.

**Additional context**
Used this simple repo as example: https://github.com/marchesir/learn-k8s which has this simple .devcontainer.json (https://github.com/marchesir/learn-k8s/blob/main/.devcontainer.json) but ignores it and uses its own .devcontainer.json.

![dc1](https://user-images.githubusercontent.com/10289630/163842255-e47c3a43-54ec-4de8-ba17-eb9a41eeaeb7.jpg)

![dc2](https://user-images.githubusercontent.com/10289630/163842257-add4a44c-fd42-49c3-82c3-ba22fd455687.jpg)

**Comment by btmurrell** [comment]
Yes, I wonder why that would not be the obvious choice. I was quite surprised to try this feature to see that it ignored my devcontainer.json file... +100 for me.
